test/test_basic.py

```diff
@@ -105,10 +105,14 @@ def newSession(self):
         # API URL is on the container host's localhost; translate for the container DNS
         websocket_url = self.api_url.replace('localhost', 'host.containers.internal').replace('https:', 'wss:')
         podman = ['podman', 'run', '-d', '--pod', 'webconsoleapp',
+                  '--volume', './3scale/certs/ca.crt:/usr/local/share/ca-certificates/3scale-ca.crt:ro',
+                  # in production, the bridge connector gets sent to target system via Ansible
+                  '--volume', './server:/server:ro',
                   '--network', 'consoledot', 'localhost/webconsoleserver']
-        cmd = ['websocat', '--basic-auth', 'admin:foobar', '-b', '-k',
-               f'{websocket_url}{config.ROUTE_WSS}/sessions/{sessionid}/ws',
-               'cmd:cockpit-bridge']
+        cmd = ['sh', '-exc',
+               f'update-ca-certificates; '
```

Comment:
You can get rid of `update-ca-certificates` by loading additional trust anchors into the SSL context.

Reply:
I'm fine with adding `--extra-ca-cert`, but I don't want to use it during tests. In production, we expect consoledot to have a generally known valid certificate, and will (presumably) *not* install any custom CA. So I'd like to make sure that stuff works without this extra flag.

Reply:
Actually, I'm not entirely sure whether I'm fine with adding that. This should be as minimal as possible, and not have features that we don't test/use. But anyway, I need to fix that test race condition first, then I'll be less grumpy and will probably add it :grin: